PSCoreScripts/Copy-SitefinityDatabase.ps1

```diff
@@ -42,7 +42,7 @@ if (!$SqlServerResource) {
 
 # extract the build number if it is not provided
 if (!$PSBoundParameters.ContainsKey("ReleaseNumber")) {
-    if ($ReleaseNumber = $ENV:RELEASE_RELEASENAME) {
+    if ($ReleaseNumber -eq $ENV:RELEASE_RELEASENAME) {
```

Comment:
this fixes a PSScriptAnalyzer error

Reply:
The line should say
> if ($ENV:RELEASE_RELEASENAME) { 
ie testing if the environment variable exists before trying to split it

Reply:
if() condition has been updated, and a test for it has been added.